index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Daniels Portfolio</title>
+</head>
+<body>
+    <nav>
+        <ul class="buger">
+
+            <li class="Mi">Daniel</li>
+           <li class="ham">
+            <i class="fa fa-bars"></i> 
+           </li>  
+        </ul>
+    </nav>
```

Comment:
- Kindly, I suggest you make sure that the container of the header section is a `header` tag.
- Kindly, make sure the logo is inside a link `a` tag.
- [OPTIONAL] Please, make sure you have descriptive `classes` in your code base to follow HTML best practices and not by following that your CSS file will be easy to read by another developer and code reviewer, and your project will look more professional.